projects/Amlogic/bootloader/release

```diff
@@ -3,7 +3,5 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
-. config/options $1
-
 mkdir -p $RELEASE_DIR/3rdparty/bootloader
-cp -a $INSTALL/usr/share/bootloader/* $RELEASE_DIR/3rdparty/bootloader
+  cp -a $INSTALL/usr/share/bootloader/* $RELEASE_DIR/3rdparty/bootloader
```

Comment:
Stray indent?

Reply:
Not in this case, it's consistent with the existing `release` files where we indent-within-mkdir

https://github.com/LibreELEC/LibreELEC.tv/blob/30ced21a40bd47ad86261812418c577277161b59/projects/Rockchip/bootloader/release

https://github.com/LibreELEC/LibreELEC.tv/blob/30ced21a40bd47ad86261812418c577277161b59/packages/tools/bcm2835-bootloader/release

See coding standards RFC: https://github.com/LibreELEC/LibreELEC.tv/pull/3055/files#diff-27d64c693629212f25ac5e5c03c2e2a2R29